.gitignore

```diff
@@ -1,4 +1,5 @@
 .vscode
+.idea
```

Comment:
curious, what is a .idea file ?

Reply:
It's just coming from Jetbrains IDE's basically the same as .vscode, ide settings.